### PR TITLE
Fixes #3458 - remove optional GitHub username field

### LINKED ISSUE #3458: Remove "Optional GitHub username" field
Lets remove "Optional GitHub username" field and make it so the issue submitted upon pressing "Report anonymously"

### PR DESCRIPTION
**Pull request by ksy36**
r? @karlcow @miketaylr

### CODE REVIEW

**Review by karlcow — APPROVED** [review]
I searched and didn't find any related tests. Oops. 
We had before. :) I guess we don't need anymore. 

Thanks @ksy36

File: webcompat/static/js/lib/wizard/steps/submit.js [hunk 1]
```diff
@@ -8,31 +8,16 @@
 
 import $ from "jquery";
 import { showContainer } from "../ui-utils.js";
-import { isGithubUserNameValid } from "../validation.js";
```
Comment (miketaylr) [inline webcompat/static/js/lib/wizard/steps/submit.js:11]: Do we want to delete `isGithubUserNameValid` as well? I don't think it's used anywhere else.

Comment (ksy36) [reply webcompat/static/js/lib/wizard/steps/submit.js:11]: Good point, missed that!

Comment (ksy36) [reply webcompat/static/js/lib/wizard/steps/submit.js:11]: ready for another look @miketaylr :)

**Pull request merged**